google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionClientTest.java

```diff
@@ -273,7 +273,7 @@ public void batchCreateSessionsWithExceptions() {
       final List<Long> errorOnChannels = new ArrayList<>();
       if (behavior == AddRemoveSetException.REMOVE) {
         for (int c = 0; c < spannerOptions.getNumChannels(); c++) {
-          errorOnChannels.add(Long.valueOf(c));
+          errorOnChannels.add((long) c);
```

Comment:
I'm actually not sure whether I agree with the (presumably automatically proposed) change here. The goal is to remove unnecessary boxing, but `List<Long>#add(Long e)` expects a `Long` and not a `long`, so I would say that the boxing is not unnecessary here. Or am I missing the point of this change?

Reply:
That is true, I think it wants to avoid manual boxing of values. Let me know if you have strong opinions about this and I can change it back, otherwise I will leave it as is.

Reply:
No, if this removes a warning from the static code checker, then I'm happy with it.